armada-ths/ais
# Implemented slider questions in API.

Implemented grading questions and work fields in api/questions/. Also removed fair fields from questions, as Survey already contains that reference.

To test:
* make sure you have a matching survey for current fair
* create at least 1 grading quesetion
* (optional) create at least 1 slider question
* make sure the questions correspond to the survey
* create at least 1 work field area (you can combine this with the next step)
* create at least 1 work field (you can combine this with both previous and following steps)
* make sure the work fields correspond to the survey
* goto localhost:8000/api/questions/
* and make sure you see a valid json response with your questions and areas.
